rs/types/management_canister_types/src/lib.rs

```diff
@@ -939,73 +853,12 @@ impl TryFrom<pb_canister_state_bits::LogVisibilityV2> for LogVisibilityV2 {
                         })
                     })
                     .collect::<Result<Vec<PrincipalId>, _>>()?;
-                Ok(Self::AllowedViewers(BoundedAllowedViewers::new(data)))
+                Ok(Self::AllowedViewers(BoundedAllowedViewers::new(principals)))
             }
         }
     }
 }
 
-#[test]
-fn test_log_visibility_v2_serialization_roundtrip() {
```

Comment:
Can't we keep this test? It seems to not depend on the old `LogVisibility` at all.

Reply:
After removing the only special case of turning `AllowedViewers` to the default `Controllers` there is no point in having such a test.
It will just check that all the enum variants are serialized and deserialized correctly which is expected by adding `CandidType` and implementing `Payload` trait. 
Additionally there is no other type or enum that has such a test in this file.
So I'd rather keep it removed.